src/quadmath.hpp

```diff
@@ -32,7 +32,7 @@ static inline __float128 strtoflt128(const char *s, char **sp) {
 
 static inline int quadmath_snprintf(char *s, size_t size, const char *format, ...) {
     va_list args;
-    va_start(format, args);
+    va_start(args, format);
```

Comment:
I had to switch this for MacOS so I think we need `#if defined(__APPLE__)` here.

Reply:
this implementation is inside a `#if defined(_MSC_VER)`...how would it be compiled for apple?

Reply:
hmm good point. I don't know what happened that made me think we needed to switch those args. 

I'll go ahead and merge this, the fix with unreachable is good and the code I'm currently working on removes quadmath dependency anyway.